src/sessions/p2p_session.rs

```diff
@@ -913,14 +924,17 @@ impl<T: Config> P2PSession<T> {
                         for remote in self.player_reg.remotes.values_mut() {
```

Comment:
I keep getting the following panic right after peers connect in my `bevy_ggrs` project.

```
thread 'main' panicked at 'cell not found!: frame 100', ggrs-c119670c0a115c50/94604ce/src/sessions/p2p_session.rs:921:44
```

The frame not found seems to depend on the specified desync interval.

```
.with_desync_detection_mode(DesyncDetection::On { interval: 100 })
```

Changing the max prediction window and max frames behind didn't do much either

```
.with_max_prediction_window()
.with_max_frames_behind()
```

Is there anything that has to be configured in order for this PR to properly work in
exisiting `bevy_ggrs` projects, or this not expected to work with `bevy_ggrs` at all yet?